src/JustBehave.Tests/JustBehave.Tests.csproj

```diff
@@ -3,28 +3,23 @@
   <Import Project="..\JustBehave.Common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>net452</TargetFrameworks>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <DefineConstants>TRACE;CODE_ANALYSIS;</DefineConstants>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AutoFixture" Version="3.40.0" />
-    <PackageReference Include="AutoFixture.AutoRhinoMocks" Version="3.40.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
-    <PackageReference Include="NLog" Version="4.2.3" />
-    <PackageReference Include="NUnit" Version="3.0.1" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0" />
-    <PackageReference Include="RhinoMocks" Version="3.6.1" />
-    <PackageReference Include="Shouldly" Version="2.8.3" />
-    <PackageReference Include="xunit" Version="2.1.0" />
-    <PackageReference Include="xunit.abstractions" Version="2.0.0" />
-    <PackageReference Include="xunit.assert" Version="2.1.0" />
-    <PackageReference Include="xunit.core" Version="2.1.0" />
-    <PackageReference Include="xunit.extensibility.core" Version="2.1.0" />
-    <PackageReference Include="xunit.extensibility.execution" Version="2.1.0" />
+    <PackageReference Include="AutoFixture" Version="4.0.0-rc1" />
+    <PackageReference Include="AutoFixture.AutoNSubstitute" Version="4.0.0-rc1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="NLog" Version="4.5.0-beta07" />
+    <PackageReference Include="NUnit" Version="3.8.1" />
```

Comment:
How come there’s NUnit and xunit?

Reply:
Because it supports both frameworks, although it doesn't appear to use any xunit dependency directly. See #23